src/pages/Search/Search.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+import SearchResult, {
+  SearchResultProps,
+} from '../../components/SearchResult/SearchResult';
+import styles from './Search.module.css';
+import Overlaymenu from '../../components/OverlayMenu/OverlayMenu';
+
+function Search(): JSX.Element {
+  const [results, setResults] = useState<SearchResultProps[] | null>([]);
+  // const [search, setSearch] = useState('');
+  const [query, setQuery] = useState<string | undefined>(undefined);
```

Comment:
I don't understand why query can be undefined: If you do not put a value in the input field, isn't the value an empty string?